packages/jest-mock/src/index.ts

```diff
@@ -73,25 +73,12 @@ export type MaybeMockedDeep<T> = T extends MockableFunction
   : T;
 
 export type MaybeMocked<T> = T extends MockableFunction
-  ? MockedFunction<T>
+  ? MockedFn<T>
   : T extends object
   ? MockedObject<T>
   : T;
 
 export type ArgsType<T> = T extends (...args: infer A) => any ? A : never;
-export type Mocked<T> = {
```

Comment:
this does not make much sense here so  i am removing this

Reply:
These are exported types, so removing them seems to be breaking. This is subjective, but I find it a better pattern to use utility types for type transformations instead of no-op functions. Unfortunately these are left undocumented (and somewhat incomplete), but no doubt they make sense.

Reply:
breaking to remove yes, but that can be done for Jest 28. But _why_ do they "not make much sense"?

Reply:
why i keep forgetting that changes can break builds . not sure about `Mocked` 🙈 but removing `MockedClass` make [some sense ](https://github.com/facebook/jest/blob/main/docs/MockFunctionAPI.md#jestmockedclass). No ?

Reply:
I'm reverting this change . Thanks for responding.